explore-analyze/visualize/charts/bar-charts.md

```diff
@@ -0,0 +1,413 @@
+4. Configure the reference line's appearance:
+   - **Name**: The name of the reference line. 
+   - **Value format**: Controls how any label for the line is displayed (default formatting or other available numeric/date formats).
+   - **Text decoration**: Selects whether the label shows nothing or the line’s name.
+   - **Icon decoration**: Allows adding an icon next to the label.
+   - **Line**: Sets stroke width (1 px) plus pattern (solid or dashed).
+   - **Fill**: Decides whether to shade the area above or below the reference line. None leaves the area unfilled.
+   - **Color**: Sets the line (and fill) color.
+
+### Add annotations [add-annotations]
+
+```{applies_to}
+stack: preview
+serverless: preview
+```
+
+Annotations are a powerful way to add context to your visualizations by marking important events directly on your charts, helping viewers understand the relationship between your data and real-world events. They appear as vertical lines or bands across your visualization, making it easy to correlate data patterns with known events.
+
+::::{tip}
+Annotations are great for:
+- Marking deployment events
+- Highlighting system maintenance windows
+- Indicating configuration changes
+- Marking business events that might impact metrics
+::::
+
+To add an annotation:
+
+1. Create a **Bar** chart visualization.
+2. Click **Add layer**, then select the **Annotations** layer type. Create a new annotation or select an existing one from your library.
+3. For new annotations, select a {{data-source}}. It can be any {{es}} index containing timestamp data.
+4. Click the newly-created annotation to edit it. 
+5. Select the annotation's placement. It can be a static date that you choose or a based on a custom query. Choose a static date if you want to manually mark a specific point on the bar chart. Choose a query-based annotation to dynamically mark events based on data. 
+6. Customize how the annotation appears:
+   - **Name**: The name of the annotation. 
+   - **Icon decoration**: Allows adding an icon next to the label.
+   - **Text decoration**: Selects whether the label shows nothing or the annotation's name.
+   - **Line**: Sets stroke width (1 px) plus pattern (solid or dashed).
+   - **Color**: Sets the line color.
+   - **Hide annotation**: Hides or displays the annotation. By default, this setting is turned off and the annotation is displayed.
+   - **Tooltip**: Adds additional fields to the annotation label. 
+6. Click **Close** to apply the configurations. Your bar chart will now display the annotations at the appropriate time points.
+
+To edit existing annotations click on them in the annotations list. If you no longer need an annotation, you can delete it.
+
+## Bar chart settings [settings]
+
+Customize your bar chart to display exactly the information you need, formatted the way you want.
+
+### Horizontal axis settings [horizontal-axis-options]
```

Comment:
Maybe to do in a follow up but those horizontal axis, vertical axis, and breakdown settings look identical for the various time series charts, we could create a snippet that we reuse